habitat_baselines/rl/ver/queue.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import TYPE_CHECKING
+
+try:
+    import faster_fifo
+
+    use_faster_fifo = True
+except ImportError:
+    use_faster_fifo = False
+
+
+if not TYPE_CHECKING and use_faster_fifo:
+    import faster_fifo_reduction  # noqa: F401
+
+    BatchedQueue = faster_fifo.Queue
+else:
+    import multiprocessing.queues
+    import queue
+    import time
+    import warnings
+
+    warnings.warn(
+        "Unable to import faster_fifo."
+        " Using the fallback. This may reduce performance."
+    )
+
+    class BatchedQueue(multiprocessing.queues.Queue):
```

Comment:
Shouldn't you extend from torch.multiprocessing module if it's available? @erikwijmans

Reply:
I'll change it to that. `torch.multiprocessing.Queue` is just an alias, but best to be safe.